app/controllers/users_controller.rb

```diff
@@ -30,7 +38,7 @@ def destroy
   private
 
   def user_params
-    params.require(:user).permit(:name, :email, :password, :uid)
+    params.require(:user).permit(:name, :email, :password, :admin, :uid)
```

Comment:
I would suggest not allowing `:admin` to be passed as a param. Because then every user could hack the system and set themselves as admin.

Once again, I would suggest allowing this only under an admin namespaced in admin controllers.

Reply:
Good point, sounds like admin namespacing for the routes makes sense, done in sha: 1ea0d5e